src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier randomFigures = new FigureSupplier();
+        Figure[] array = new Figure[6];
+        for (int i = 0; i < 6; i++) {
+            if (i < 3) {
+                array[i] = randomFigures.getRandomFigure();
+            } else {
+                array[i] = randomFigures.getDefaultFigure();
+            }
+            array[i].getArea();
+            array[i].draw();
+        }
```

Comment:
1. It's better to use constant and relational expression in order to make change in one place instead of 3 in case we want to change amount of figures
2. usage of array[i].getArea() is meaningless, returned value is not used anywhere

```suggestion
    private static final int FIGURES_AMOUNT = 6

    public static void main(String[] args) {
        FigureSupplier randomFigures = new FigureSupplier();
        Figure[] array = new Figure[FIGURES_AMOUNT];
        for (int i = 0; i < FIGURES_AMOUNT; i++) {
            if (i < FIGURES_AMOUNT / 2) {
                array[i] = randomFigures.getRandomFigure();
            } else {
                array[i] = randomFigures.getDefaultFigure();
            }
            array[i].draw();
        }
```